doc/sphinx/practical-tools/coq-commands.rst

```diff
@@ -124,11 +124,11 @@ and ``coqtop``, unless stated otherwise:
 
      :ref:`names-of-libraries` and the
      command Declare ML Module Section :ref:`compiled-files`.
-:-Q *directory* dirpath: Add physical path *directory* to the list of
+:-Q *directory* *dirpath*: Add physical path *directory* to the list of
   directories where |Coq| looks for a file and bind it to the logical
   directory *dirpath*. The subdirectory structure of *directory* is
   recursively available from |Coq| using absolute names (extending the
-  dirpath prefix) (see Section :ref:`qualified-names`).Note that only those
+  :n:`@dirpath` prefix) (see Section :ref:`qualified-names`).Note that only those
```

Comment:
You can use ``:token:`dirpath` `` here, I think.
Also, there's a missing space before `Note that only…`

Reply:
> You can use ``:token:`dirpath` ``here, I think.

In the same file, there are plenty of ``:n:@`foo` `` and no ``:token:`foo` ``. It seems better to me to keep the file consistent in style.

>  Also, there's a missing space before `Note that only`…

Thanks, I should have noticed it in passing. Fixed.